frontend/yarn.lock

```diff
@@ -1910,18 +1910,20 @@
   dependencies:
     http-proxy-middleware "^1.0.6"
 
-"@nuxtjs/pwa@3.2.0":
-  version "3.2.0"
-  resolved "https://registry.yarnpkg.com/@nuxtjs/pwa/-/pwa-3.2.0.tgz#23b72a3a940b941febd87dbbd98ec48f292b6b0e"
-  integrity sha512-IpzVC5WrP6kQU+gwe2vBjB9xRZlQprhlB1a2nh6SrD0N1IbvCq+AyiEJBDxhr/Z4887zLsRdHVSHS+o08F2oTA==
-  dependencies:
-    defu "^3.1.0"
-    execa "^4.0.3"
-    fs-extra "^9.0.1"
+"@nuxtjs/pwa@^3.3.5":
+  version "3.3.5"
+  resolved "https://registry.yarnpkg.com/@nuxtjs/pwa/-/pwa-3.3.5.tgz#db7c905536ebe8a464a347b6ae3215810642c044"
+  integrity sha512-8tTmW8DBspWxlJwTimOHTkwfkwPpL9wIcGmy75Gcmin+c9YtX2Ehxmhgt/TLFOC9XsLAqojqynw3/Agr/9OE1w==
+  dependencies:
+    clone-deep "^4.0.1"
+    defu "^3.2.2"
+    execa "^5.0.0"
+    fs-extra "^9.1.0"
```

Comment:
We can't upgrade this dependency. It will break the PWA

- https://github.com/mealie-recipes/mealie/issues/1077
- https://github.com/nuxt-community/pwa-module/issues/528